server/personality/dto/create-personality.dto.ts

```diff
@@ -11,6 +11,18 @@ export class CreatePersonality {
     @IsString()
     wikidata: string;
 
+    @IsString()
```

Comment:
As discussed in Slack. Let's remove the Wikidata integration for now. The idea is to just confirm that we can tag a personality and we can implement that in a follow-up patch.